docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -125,6 +131,9 @@ public static void Example()
             IEnumerable<ImageData> testImages = LoadImagesFromDirectory(
                 imagesForPredictions, false);
 
+            byte[] imgBytes = File.ReadAllBytes(testImages.First().ImagePath);
+            VBuffer<Byte> imgData = new VBuffer<byte>(imgBytes.Length, imgBytes);
```

Comment:
It doesn't look like this in-memory image is being used for the Prediction when using the PredictionEngine, right?
In fact, looks like this object 'imgData' is not used later on?

We need to have and test Predictions by providing the in-memory image instead of file paths.
I cannot find such a code.